### PR TITLE
Update Learning-Environment-Executable.md

### PR DESCRIPTION
**Pull request by unityjeffrey**
fixed issue when creating a build folder in the assets folder.  Refering to https://github.com/Unity-Technologies/ml-agents/issues/2033

### CODE REVIEW

File: docs/Learning-Environment-Executable.md [hunk 1]
```diff
@@ -57,9 +57,9 @@ launches our environment executable. This means:
    * In the File dialog, navigate to your ML-Agents directory.
    * Assign a file name and click **Save**.
    * (For Windows）With Unity 2018.1, it will ask you to select a folder instead
-     of a file name. Create a subfolder within the ML-Agents folder and select
+     of a file name. Create a subfolder within the root directory and select
      that folder to build. In the following steps you will refer to this
-     subfolder's name as `env_name`.
+     subfolder's name as `env_name`. You cannot create builds in the Assets folder
```
Comment (xiaomaogy) [inline docs/Learning-Environment-Executable.md]: Change "cannot" to "shouldn't"? You can but you shouldn't.

**Comment by xiaomaogy** [comment]
Looks good to me.

File: docs/Learning-Environment-Executable.md (hunk 1 above)
Comment (unityjeffrey) [reply docs/Learning-Environment-Executable.md]: you actually cannot - it will throw an error

Comment (xiaomaogy) [reply docs/Learning-Environment-Executable.md]: Oh ok, then nvm.

**Pull request merged**